build-tools/packages/build-cli/src/commands/check/latestVersions.ts

```diff
@@ -0,0 +1,82 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { Args } from "@oclif/core";
+import * as semver from "semver";
+import { findPackageOrReleaseGroup, packageOrReleaseGroupArg } from "../../args.js";
+import { BaseCommand, sortVersions } from "../../library/index.js";
+
+export default class LatestVersionsCommand extends BaseCommand<typeof LatestVersionsCommand> {
+	static readonly summary =
+		"Determines if an input version matches a latest minor release version.";
+
+	static readonly description =
+		"This command is used in CI to determine if a pipeline was triggered by a release branch with the latest minor version of a major version.";
+
+	static readonly args = {
+		version: Args.string({
+			required: true,
+			description: "The version corresponding to the pipeline trigger branch.",
+		}),
```

Comment:
This might be a good opportunity to use `Args.custom<SemVer>` instead. The docs don't have an example for args, but they do for Flags, and it's very similar. There may be examples in our repo too. Anyway, the benefit is that you can verify the string is a valid semver string at argument validation time, and your code in run will receive a SemVer object instead of a string to work with.